frame.go

```diff
@@ -401,7 +401,9 @@ func (f *Frame) Close() error {
 
 	// Close all views.
 	for _, view := range f.views {
-		_ = view.Close()
+		if err := view.Close(); err != nil {
```

Comment:
@benbjohnson did not recall if this was intentional and thought we should address it.